cloud_Azure/terraform/module/network_watcher.tf

```diff
@@ -1,32 +1,62 @@
-# Creates network watcher
-resource "azurerm_network_watcher" "kentik_network_watcher" {
-  name                = "${var.prefix}_network_watcher"
-  location            = var.location
-  resource_group_name = var.resource_group_name
+# Created automatically by Azure when VirtualNetwork is created or updated in the subscription, see:
```

Comment:
What is created automatically? This resource file?

Reply:
Updating text to:
"Network Watcher is created automatically by Azure when VirtualNetwork is created or updated in the subscription"